README.md

```diff
@@ -19,14 +38,13 @@ Please [submit an issue](https://github.com/smartface/sample-clienteling/issues)
 
 ## Support & Documentation & Useful Links
 - [Guides](https://developer.smartface.io)
-- [API Docs](http://docs.smartface.io)
```

Comment:
docs.smartface.io also works and redirects to same source, but it's fine

Reply:
Well. I copied and pasted those links from other sample projects of Smartface in github. I thought they were carefully checked and committed before. But do not worry. I will check and correct them all. Many thanks for your advises.